### DISCUSSION TITLE
Beginner Help

### DISCUSSION THREAD

**Discussion by SturnES** [Mezryss/FVTT-Genesys#50] [Q&A] [answered]
Some notes I made for myself that others may find useful.

### NUMBER ONE RULE

If you can't figure it out, you probably need to create an Item (to right menu bar of Foundry VTT) then drag it someplace. Actors need Items drug to them (Archetypes, Careers, Talents, Skills, Equipment). Likewise, Equipment need Items drug to them (Qualities, Skills).

### STARTING WITH SKILLS

While Skills can be created as Items, it is simpler to use _Compendium Packs > Core Skills Compendium_. The work has already been done for you.

But you have custom skills? Make a copy of the Core Skills compendium first (_right click > Duplicate Compendium_). Then, you can edit skills, delete skills, or add skills.

How do you get your new skills list to show up in your Actor sheets? Go to _Configure Settings > Genesys_. Under the first section you will see _Skills Compendium_ with the default entry of _genesys-skills.crb_. This needs to be changed to _world.insertyourcompendiumname_. Before you give it a go, the syntax is important and can be tricky. Spaces need replaced with a dash (-) and capitalization is removed. So, if you named your new skills compendium _Mad Skills_, then enter _world.mad-skills_.

It's important to note that Player Actor sheets created before implementing your new skills compendium won't be affected or changed. You will need to recreate them to use the new skills list.

### WORKFLOW AFTER SKILLS COMPENDIUM

These are just suggestions but may help you get a grasp of what you need to do. To avoid not having the right Item data ready, create Items in this order:

1. Skills or Skills Compendium
2. Item Qualities
3. Equipment (Armors, Consumable, Container, Gear, or Weapons)
4. Archetypes, Careers, and Talents.
5. Player Actor Sheet (since it will need all of the above)_

Why? Some of the Items need a previous Item to be completed. For example, if you are creating an Axe (from the Terrinoth sourcebook) it has the Vicious quality. You will need the Vicious Item Quality already in place to drag to the Axe Weapon Item to complete it.

Here is what each Item _may_ need completed first:

**Armor:** Item Qualities (not always, most Armor won't have qualities).
**Consumable**: Nothing, no dependencies.
**Container:** Nothing, no dependencies.
**Gear:** Nothing, no dependencies.
**Item Quality:** Nothing, no dependencies.
**Weapon:** Skills and Item Qualities
**Special Ability:** Nothing, no dependencies.
**Archetype:** Granted equipment (Armor, Consumable, Container, Gear) if needed.
**Career:** Skills
**Talent:** Nothing, no dependencies.

This is for creating Player Character Actors which need lots of other Items to create them. If you are creating Adversary's
(Minion, Rival, Nemesis), you will need only relevant Skills, Talents, Special Abilities, and Equipment first.

**Adversaries:** Skills, Talents, Special Abilities, Equipment

Wait?! You're telling me I have to create ALL item qualities, equipment, skills, talents, archetypes, and careers before I
make my first actor! No, not all, just the Skills Compendium and which of the Items (above) you need for the specific
character or adversary. For example, if you are creating the Wolf Minion from the Expanded Player's Guide, you will need the Skills Compendium completed, a Special Ability item of "Pack Tactics", and a Weapon item of "Wolf Bite" to drag over to your Actor Minion sheet.

### A NOTE ON CREATING ADVERSARIES

If you are copying data over from another source into a new adversary Actor (Minion, Rival, Nemesis) consider that any
Weapon Item you drag into it will bring along the Brawn damage adjustment characteristic with it. For example, if you
are copying the Wolf Minion from the Expanded Player's Guide, it has a Bite attack of 5 damage and Brawn 2. The Brawn damage adjustment has already been considered. Thus if you create a Weapon Item of "Wolf Bite" and type in "5" for its damage, when you drag it to the Wolf Minion sheet, it's going to end up listing it as 7 damage since it will automatically add the Brawn of the Wolf again. So, you could enter the Weapon of Wolf Bite as 3 damage with the Brawn characteristic selected or you could simply choose no characteristic and enter 5 damage. Either way will end up with 5 damage on your Wolf Minion sheet.

### DICE CODES

How do you put special dice symbols or dice symbols into your journals or text fields? Use the codes below. The letters in the brackets [ ] represent a die or die symbol such as an "s" for success. Note that for multiple symbols or dice, you can insert multiple single letters. For example, to make three difficulty dice symbols together for a Hard difficulty check, type _@die[ddd]_. No need to type _@die[d]_ three times.

**Symbols: @sym[satfhd]** (Alternatively, @symbol[], @symbols[]) 

**Dice: @die[apbdcs]** (Alternatively, @dice[], and you can use either 'i' or 'd' for difficulty dice) 

**Dice by color:** @die[color,gybprk] (Alternatively, @dice[], also instead of 'col' it can be 'color' or 'colors')

For ease of entry, you can copy & paste the entry below into a document such as Notepad for easy later copying & pasting into your text fields.

```
**SUCCESS**
@sym[s]

**ADVANTAGE**
@sym[a]

**TRIUMPH**
@sym[t]

**FAILURE**
@sym[f]

**THREAT**
@sym[h]

**DESPAIR**
@sym[d]

**ABILITY**
@die[a]

**PROFICIENCY**
@die[p]

**BONUS**
@die[b]

**DIFFICULTY**
@die[d]

**CHALLENGE**
@die[c]

**SETBACK**
@die[s]
```

**Comment by bartvdbraak** [comment, marked as answer]
Fantastic job sharing these useful notes for beginners! 

Your step-by-step approach, emphasis on item creation order, and tips on customizing skills are invaluable for a smooth gameplay experience. Thank you for taking the time to compile and share this valuable resource with the community. Your contribution is greatly appreciated!

**Comment by SturnES** [comment]
If you want to turn display of the Story Pool on or off for some reason you can use the macro below. There are modules to hide the UI for a better battlemap display, but it doesn't remove the Story Pool.

Create a Macro. Macro's are along the bottom of Foundry VTT, just click on an empty number to begin. Select "script" from the options, then type in the following perfectly:

`$(".story-point-tracker").toggle()`

When you execute this Macro, the Story Pool boxes on the bottom right of the display will be hidden or revealed.

**Reply by Assembling-Kings** [reply]
Or simply `$(".story-point-tracker").toggle()` which should toggle between both states.

**Reply by SturnES** [reply]
I tested and updated it, thanks!

**Comment by piratepressllc** [comment]
how do I increase the characters melee or ranged defense on the character sheet. For instance, I created a sword item and gave it the defensive quality (also a separate item) that adds 1 defense to the character. I don't see an option to select that +1 defense on the effects tab?

**Reply by josht-dev** [reply]
> how do I increase the characters melee or ranged defense on the character sheet. For instance, I created a sword item and gave it the defensive quality (also a separate item) that adds 1 defense to the character. I don't see an option to select that +1 defense on the effects tab?

You can do it by using the Foiundry [active effects](https://foundryvtt.com/article/active-effects/). Add a passive effect to the defensive item quality and set the `CHANGES`  tab to the following:  ` Updated for version 2.18`
Attribute Key = `Melee Defense` and/or `Ranged Defense`
Change Mode = `Add`
Value = `1`

**Comment by noblue123** [comment]
Is there a way to roll dice without an actor sheet open? Also, is there a way to conceal rolls from the players as the GM? I noticed toggling it to private GM rolls does not stop the players from seeing the roll.

**Reply by josht-dev** [reply]
I'm not sure in version 13. Version 12 I believe had a way to roll without an actor sheet. 

My personal opinion on private rolls  is that with the genesys narrative dice system, they should not be a thing as players should have the option to help narrate the dice results. I don't think private rolls are even ever mentioned in the rulebook.

**Reply by Assembling-Kings** [reply]
In FVTTv12 you can click on the dice icon on the chat sidebar to bring the dice roller, while in FVTTv13 you can use the chat command `/gendr`.

Also, rolls respect the roll mode they've done under so it should work. Or are you saying that you want to roll without a message showing that dices were rolled privately? If that is what you want then no, there is no way to do whisper rolls currently.
